test/integration/basic_nsmdp_test.go

```diff
@@ -0,0 +1,65 @@
+// +build basic
+
+package nsmd_integration_tests
+
+import (
+	"github.com/networkservicemesh/networkservicemesh/test/integration/nsmd_test_utils"
+	"github.com/networkservicemesh/networkservicemesh/test/kube_testing"
+	"github.com/networkservicemesh/networkservicemesh/test/kube_testing/pods"
+	. "github.com/onsi/gomega"
+	"testing"
+)
+
+func TestNSMDDP(t *testing.T) {
+	RegisterTestingT(t)
+
+	if testing.Short() {
+		t.Skip("Skip, please run without -short")
+		return
+	}
+
+	k8s, err := kube_testing.NewK8s()
+	defer k8s.Cleanup()
+
+	Expect(err).To(BeNil())
+
+	k8s.Prepare("nsmd", "icmp")
+
+	nodes := nsmd_test_utils.SetupNodesConfig(k8s, 1, defaultTimeout, []*pods.NSMDPodConfig{})
+	icmpPod := nsmd_test_utils.DeployICMP(k8s, nodes[0].Node, "icmp-responder-nse1", defaultTimeout)
```

Comment:
Please use `icmp-responder-nse-1` with a dash before th number. We had #817 treating this across all tests.